tests/message/unittest_message_id_store.py

```diff
@@ -1,14 +1,18 @@
 # Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
 # For details: https://github.com/PyCQA/pylint/blob/main/LICENSE
 
+from os.path import abspath, dirname, join
```

Comment:
```suggestion
from pathlib import Path

EMPTY_FILE = str(Path(__file__).parent.parent.resolve() / "regrtest_data" / "empty.py")
```
I don't want to be ideological about but pathlib.Path is a really nice class 😛